src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -245,6 +269,24 @@ export default {
 		if (expCookieSignifier === 'b') {
 			this.$kvTrackEvent('Borrower Profile', 'EXP-GROW-655-Aug2021', expCookieSignifier);
 		}
+		// EXP-MARS-95-May2022
+		this.apollo.watchQuery({ query: publicSocialShareQuery }).subscribe({
+			next: ({ data }) => {
+				return getExperimentSettingAsync(this.apollo, socialShareExpKey)
```

Comment:
The publicSocialShareQuery already fetches the experiment setting, so this call is redundant. Instead, this should be replaced with a call to get the experiment version:
```js
import experimentAssignment from '@/src/graphql/query/experimentAssignment.graphql';
...
return this.apollo.query({
	query: experimentAssignment,
	variables: { id: socialShareExpKey },
}).then(versionResult => {
	const version = versionResult?.data?.experiment?.version;
	if (version) {
		trackExperimentVersion(
			this.apollo,
			this.$kvTrackEvent,
			'Borrower Profile',
			socialShareExpKey,
			'EXP-MARS-95-May2022'
		);
		this.showSocialShareBlock = version === 'b';
	}
});
```

Reply:
Thanks @emuvente, I'll keep this in mind